DeviceDiagnostics/DeviceDiagnostics.cpp

```diff
@@ -264,14 +259,14 @@ namespace WPEFramework
         uint32_t DeviceDiagnostics::getVideoDecoderStatus(const JsonObject& parameters, JsonObject& response)
         {
             LOGINFOMETHOD();
-            response["video"] = getMostActiveDecoderStatus("video");
+            response["videoDecoderStatusChange"] = getMostActiveDecoderStatus("video");
             returnResponse(true);
         }
 
         uint32_t DeviceDiagnostics::getAudioDecoderStatus(const JsonObject& parameters, JsonObject& response)
         {
             LOGINFOMETHOD();
-            response["audio"] = getMostActiveDecoderStatus("audio");
+            response["audioDecoderStatusChange"] = getMostActiveDecoderStatus("audio");
```

Comment:
the param should be "audioDecoderStatus". The param in the event should be "audioDecoderStatusChange".